### DISCUSSION TITLE
Exporting data from a base

### DISCUSSION THREAD

**Discussion by dhpwd** [deta/cloud-docs#366] [Help] [answered]
Is it possible to export all data from a base?

**Comment by jnsougata** [comment, marked as answer]
check: https://docs.deta.sh/docs/base/sdk/#fetch-all-items-1
fetch everything inside a base using the method explained above link and then save it to a json

**Reply by dhpwd** [reply]
Ok thanks!

p.s. correct link: https://docs.deta.sh/docs/base/sdk/#fetch-all-items

**Reply by jnsougata** [reply]
you are using js... I didn't know xd
